### PR TITLE
Add codebytes to Javascripts statements entry #4155

### PR DESCRIPTION
**Pull request by Nnamdingozi**
### Description

Addition of codebytes to docs\content\javascript\concepts\statements\statements.md file. These codebytes tool will make leaning of the concepts more interactive for the learners. Comments were also added to guide the learners on how to interact with the codebyte tools.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x ] All writings are my own.
- [x ] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x ] I have performed a self-review of my own writing and code.
- [x ] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [x ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/statements/statements.md [hunk 1]
```diff
@@ -31,6 +31,26 @@ In this example, `x`, `y`, `X`, and `z`, are declared with the `var` keyword.
 
 > **Note:** JavaScript identifiers are case-sensitive. Notice that `x` is not the same as `X`.
 
+
+## codebyte Example
+
+This example shows the output values of the declared variables
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/statements/statements.md:37]:
```suggestion
This example shows the output values of the declared variables:
```

File: content/javascript/concepts/statements/statements.md [hunk 2]
```diff
@@ -31,6 +31,26 @@ In this example, `x`, `y`, `X`, and `z`, are declared with the `var` keyword.
 
 > **Note:** JavaScript identifiers are case-sensitive. Notice that `x` is not the same as `X`.
 
+
+## codebyte Example
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/statements/statements.md:35]:
```suggestion
## Codebyte Example
```

File: content/javascript/concepts/statements/statements.md [hunk 3]
```diff
@@ -31,6 +31,26 @@ In this example, `x`, `y`, `X`, and `z`, are declared with the `var` keyword.
...
+
+This example shows the output values of the declared variables
+
+```codebyte/javascript
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/statements/statements.md:39]:
```suggestion
```codebyte/js
```

File: content/javascript/concepts/statements/statements.md [hunk 4]
```diff
@@ -31,6 +31,26 @@ In this example, `x`, `y`, `X`, and `z`, are declared with the `var` keyword.
...
+// outputs 5
+var y = 6;
+console.log(y)
+// outputs 6
+var x = 4;
+console.log(x)
+// outputs 4
+var z = x + y;
+console.log(z)
+// outputs 10, x is reassigned 4
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/statements/statements.md:51]: remove the comments with the output. There's no need for these comments the readers can compile the code themselves.

File: content/javascript/concepts/statements/statements.md [hunk 5]
```diff
@@ -39,6 +59,14 @@ Expression statements, such as function calls or assignments, evaluate an expres
 // Function call
 console.log('Hello World');
 ```
+## codebyte Example
+
+This example shows a `console.log()`  function call
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/statements/statements.md:64]:
```suggestion
This example shows a `console.log()` function call:
```

File: content/javascript/concepts/statements/statements.md [hunk 6]
```diff
@@ -39,6 +59,14 @@ Expression statements, such as function calls or assignments, evaluate an expres
...
+
+```codebyte/javascript
+console.log('Hello, World!');
+// outputs Hello, World!
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/statements/statements.md:68]:
```suggestion
```

File: content/javascript/concepts/statements/statements.md [hunk 7]
```diff
@@ -39,6 +59,14 @@ Expression statements, such as function calls or assignments, evaluate an expres
 // Function call
 console.log('Hello World');
 ```
+## codebyte Example
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/statements/statements.md:62]:
```suggestion
## Codebyte Example
```

File: content/javascript/concepts/statements/statements.md [hunk 8]
```diff
@@ -48,6 +76,20 @@ x = 6;
 console.log(x);
 ```
 
+## codebyte Example
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/statements/statements.md:79]:
```suggestion
## Codebyte Example
```

File: content/javascript/concepts/statements/statements.md [hunk 9]
```diff
@@ -48,6 +76,20 @@ x = 6;
...
+
+This example shows re-assignment of value to variable `x`
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/statements/statements.md:81]:
```suggestion
This example shows the re-assignment of value to variable `x`:
```

File: content/javascript/concepts/statements/statements.md [hunk 10]
```diff
@@ -48,6 +76,20 @@ x = 6;
...
+
+```codebyte/javascript
+var x = 5;
+console.log(x)
+//outputs 5
+x = 6;
+console.log(x);
+// outputs 6
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/statements/statements.md:89]:
```suggestion
var x = 5;
console.log(x)
x = 6;
console.log(x);
```

File: content/javascript/concepts/statements/statements.md [hunk 11]
```diff
@@ -48,6 +76,20 @@ x = 6;
 console.log(x);
 ```
 
+## codebyte Example
+
+This example shows re-assignment of value to variable `x`
+
+```codebyte/javascript
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/statements/statements.md:83]:
```suggestion
```codebyte/js
```

File: content/javascript/concepts/statements/statements.md [hunk 12]
```diff
@@ -57,6 +99,20 @@ console.log(x);
 if (x > 5) {
   console.log('x is greater than 5');
 }
+```
+## codebyte Example
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/statements/statements.md:103]:
```suggestion

## Codebyte Example
```

File: content/javascript/concepts/statements/statements.md [hunk 13]
```diff
@@ -57,6 +99,20 @@ console.log(x);
...
+
+This example shows the output of an if statement
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/statements/statements.md:105]:
```suggestion
This example shows the output of an if statement:
```

File: content/javascript/concepts/statements/statements.md [hunk 14]
```diff
@@ -57,6 +99,20 @@ console.log(x);
...
+
+```codebyte/javascript
+var x = 7
+if (x > 5) {
+  console.log('x is greater than 5');
+  // outputs x is greater than 5
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/statements/statements.md:111]:
```suggestion
```

File: content/javascript/concepts/statements/statements.md [hunk 15]
```diff
@@ -57,6 +99,20 @@ console.log(x);
 if (x > 5) {
   console.log('x is greater than 5');
 }
+```
+## codebyte Example
+
+This example shows the output of an if statement
+
+```codebyte/javascript
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/statements/statements.md:107]:
```suggestion
```codebyte/js
```

File: content/javascript/concepts/statements/statements.md [hunk 16]
```diff
@@ -100,9 +197,38 @@ for (let i = 0; i < 5; i++) {
...
+
+```
 
+```js
 // Return statement
 function add(x, y) {
   return x + y;
 }
 ```
+## codebyte Example
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/statements/statements.md:222]:
```suggestion

## Codebyte Example
```

File: content/javascript/concepts/statements/statements.md [hunk 17]
```diff
@@ -100,9 +197,38 @@ for (let i = 0; i < 5; i++) {
...
+
+This code example returns the value of the add function
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/statements/statements.md:224]:
```suggestion
This code example returns the value of the add `function`:
```

File: content/javascript/concepts/statements/statements.md [hunk 18]
```diff
@@ -100,9 +197,38 @@ for (let i = 0; i < 5; i++) {
...
 ```
+## codebyte Example
+
+This code example returns the value of the add function 
+
+```codebyte/javascript
+function add(x, y) {
+  return x + y;
+}
+// call the add function with values of 2 and 4 for the arguements x and y and print the result to the console
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/statements/statements.md:230]:
```suggestion
```

File: content/javascript/concepts/statements/statements.md [hunk 19]
```diff
@@ -100,9 +197,38 @@ for (let i = 0; i < 5; i++) {
   }
   console.log(i);
 }
+```
+## codebyte Example
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/statements/statements.md:201]:
```suggestion

## Codebyte Example
```

File: content/javascript/concepts/statements/statements.md [hunk 20]
```diff
@@ -100,9 +197,38 @@ for (let i = 0; i < 5; i++) {
...
+
+This code breaks or exits the loop when  `i` equals 3 and returns the value of `i`
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/statements/statements.md:203]:
```suggestion
This code breaks or exits the loop when  `i` equals 3 and returns the value of `i`:
```

File: content/javascript/concepts/statements/statements.md [hunk 21]
```diff
@@ -81,12 +156,34 @@ while (x < 5) {
...
+}
+```
 
+```js
 // For loop
 for (let i = 0; i < 5; i++) {
   console.log(i);
 }
 ```
+## codebyte example
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/statements/statements.md:177]:
```suggestion

## Codebyte example
```

File: content/javascript/concepts/statements/statements.md [hunk 22]
```diff
@@ -81,12 +156,34 @@ while (x < 5) {
...
+
+This works similar to `while`  loop. Conventionally `let`  keyword is used to declare a variable named `i`  in `for`  loops
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/statements/statements.md:179]:
```suggestion
This works similarly to the `while`  loop. Conventionally `let`  keyword is used to declare a variable named `i`  in `for`  loops:
```

File: content/javascript/concepts/statements/statements.md [hunk 23]
```diff
@@ -81,12 +156,34 @@ while (x < 5) {
   console.log(x);
   x++;
 }
+```
+## codebyte example
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/statements/statements.md:160]:
```suggestion

## Codebyte example
```

File: content/javascript/concepts/statements/statements.md [hunk 24]
```diff
@@ -70,6 +126,25 @@ switch (x) {
     console.log('x is neither 1 nor 2');
 }
 ```
+This example shows outputs of switch statment
```
Comment (dakshdeepHERE) [inline content/javascript/concepts/statements/statements.md:129]:
```suggestion
This example shows outputs of switch statement:
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
Work on these initial requests @Nnamdingozi then I will do the second review. 🚀

File: content/javascript/concepts/statements/statements.md (hunk 3 above)
Comment (Nnamdingozi) [reply content/javascript/concepts/statements/statements.md:39]: Hi @dakshdeepHERE. Thanks a lot for your review and suggestions. I earnestly appreciate them. I am currently making the necessary changes. Please kindly confirm that this entry means that I should change all ````codebyte/javascript to codebyte/js.

**Comment by Nnamdingozi** [comment]
Hi @dakshdeepHERE, I have made the suggested changes. Kindly go through again to see if the entries reflect  al the suggested changes.

File: content/javascript/concepts/statements/statements.md [hunk 25]
```diff
@@ -31,6 +31,27 @@ In this example, `x`, `y`, `X`, and `z`, are declared with the `var` keyword.
...
+var y = 6;
+console.log(y)
+
+var x = 4;
+console.log(x)
+
+
+var z = x + y;
+console.log(z)
+
```
Comment (avdhoottt) [inline content/javascript/concepts/statements/statements.md:52]:
```suggestion
```

File: content/javascript/concepts/statements/statements.md [hunk 26]
```diff
@@ -81,12 +152,34 @@ while (x < 5) {
...
 
+```js
 // For loop
 for (let i = 0; i < 5; i++) {
   console.log(i);
 }
 ```
+## Codebyte example
+
+This works similarly to the `while`  loop. Conventionally `let`  keyword is used to declare a variable named `i`  in `for`  loops:
```
Comment (avdhoottt) [inline content/javascript/concepts/statements/statements.md:175]:
```suggestion
This works similarly to the `while`  loop. Conventionally `let` keyword is used to declare a variable named `i`  in `for`  loops:
```

File: content/javascript/concepts/statements/statements.md [hunk 27]
```diff
@@ -70,6 +122,25 @@ switch (x) {
     console.log('x is neither 1 nor 2');
 }
 ```
+This example shows outputs of switch statement:
```
Comment (avdhoottt) [inline content/javascript/concepts/statements/statements.md:125]:
```suggestion

This example shows the outputs of the switch statement:
```

File: content/javascript/concepts/statements/statements.md [hunk 28]
```diff
@@ -48,6 +75,18 @@ x = 6;
 console.log(x);
 ```
 
+## Codebyte Example
+
+This example shows re-assignment of value to variable `x`:
```
Comment (avdhoottt) [inline content/javascript/concepts/statements/statements.md:80]:
```suggestion
This example shows the re-assignment of value to variable `x`:
```

File: content/javascript/concepts/statements/statements.md [hunk 29]
```diff
@@ -31,6 +31,26 @@ In this example, `x`, `y`, `X`, and `z`, are declared with the `var` keyword.
 
 > **Note:** JavaScript identifiers are case-sensitive. Notice that `x` is not the same as `X`.
 
+
+## Codebyte Example
+
+This example shows the output values of the declared variables:
+
+```codebyte/js
```
Comment (avdhoottt) [inline content/javascript/concepts/statements/statements.md:39]:
```suggestion
```codebyte/javascript
```
Correct this for every codebyte.

File: content/javascript/concepts/statements/statements.md [hunk 30]
```diff
@@ -39,6 +59,13 @@ Expression statements, such as function calls or assignments, evaluate an expres
 // Function call
 console.log('Hello World');
 ```
+## Codebyte Example
+
+This example shows a `console.log()` function call:
+
+```codebyte/js
```
Comment (avdhoottt) [inline content/javascript/concepts/statements/statements.md:66]:
```suggestion
```codebyte/javascript
```

File: content/javascript/concepts/statements/statements.md [hunk 31]
```diff
@@ -48,6 +75,18 @@ x = 6;
 console.log(x);
 ```
 
+## Codebyte Example
+
+This example shows re-assignment of value to variable `x`:
+
+```codebyte/js
```
Comment (avdhoottt) [inline content/javascript/concepts/statements/statements.md:82]:
```suggestion
```codebyte/javascript
```

File: content/javascript/concepts/statements/statements.md [hunk 32]
```diff
@@ -57,6 +96,19 @@ console.log(x);
 if (x > 5) {
   console.log('x is greater than 5');
 }
+```
+## Codebyte Example
+
+This example shows the output of an if statement:
+
+```codebyte/js
```
Comment (avdhoottt) [inline content/javascript/concepts/statements/statements.md:104]:
```suggestion
```codebyte/javascript
```

File: content/javascript/concepts/statements/statements.md [hunk 33]
```diff
@@ -70,6 +122,25 @@ switch (x) {
     console.log('x is neither 1 nor 2');
 }
 ```
+This example shows outputs of switch statement:
+
+```codebyte/js
```
Comment (avdhoottt) [inline content/javascript/concepts/statements/statements.md:127]:
```suggestion
```codebyte/javascript
```

File: content/javascript/concepts/statements/statements.md [hunk 34]
```diff
@@ -81,12 +152,34 @@ while (x < 5) {
   console.log(x);
   x++;
 }
+```
+## Codebyte example
+
+```codebyte/js
```
Comment (avdhoottt) [inline content/javascript/concepts/statements/statements.md:158]:
```suggestion
```codebyte/javascript
```

File: content/javascript/concepts/statements/statements.md [hunk 35]
```diff
@@ -81,12 +152,34 @@ while (x < 5) {
...
 // For loop
 for (let i = 0; i < 5; i++) {
   console.log(i);
 }
 ```
+## Codebyte example
+
+This works similarly to the `while`  loop. Conventionally `let` keyword is used to declare a variable named `i`  in `for`  loops:
+
+```codebyte/js
```
Comment (avdhoottt) [inline content/javascript/concepts/statements/statements.md:177]:
```suggestion
```codebyte/javascript
```

File: content/javascript/concepts/statements/statements.md [hunk 36]
```diff
@@ -100,9 +193,37 @@ for (let i = 0; i < 5; i++) {
   }
   console.log(i);
 }
+```
+## Codebyte Example
+
+This code breaks or exits the loop when  `i` equals 3 and returns the value of `i`:
+
+```codebyte/js
```
Comment (avdhoottt) [inline content/javascript/concepts/statements/statements.md:201]:
```suggestion
```codebyte/javascript
```

File: content/javascript/concepts/statements/statements.md [hunk 37]
```diff
@@ -100,9 +193,37 @@ for (let i = 0; i < 5; i++) {
...
 // Return statement
 function add(x, y) {
   return x + y;
 }
 ```
+## Codebyte Example
+
+This code example returns the value of the add function: 
+
+```codebyte/js
```
Comment (avdhoottt) [inline content/javascript/concepts/statements/statements.md:222]:
```suggestion
```codebyte/javascript
```

File: content/javascript/concepts/statements/statements.md [hunk 38]
```diff
@@ -100,9 +193,37 @@ for (let i = 0; i < 5; i++) {
...
+function add(x, y) {
+  return x + y;
+}
+console.log(add(2, 4))
+
```
Comment (avdhoottt) [inline content/javascript/concepts/statements/statements.md:227]:
```suggestion
```

File: content/javascript/concepts/statements/statements.md [hunk 39]
```diff
@@ -70,6 +122,25 @@ switch (x) {
...
+    console.log('x is 1');
+    break;
+  case 2:
+    console.log('x is 2');
+    break;
+  default:
+    console.log('x is neither 1 nor 2');
+}
+```
+Change the values of x to different numbers and click on run button to see their outputs.
```
Comment (avdhoottt) [inline content/javascript/concepts/statements/statements.md:142]:
```suggestion

Change the values of x to different numbers and click on the run button to see their outputs.
```
Give a line space before and after a code block.

File: content/javascript/concepts/statements/statements.md [hunk 40]
```diff
@@ -57,6 +96,19 @@ console.log(x);
 if (x > 5) {
   console.log('x is greater than 5');
 }
+```
+## Codebyte Example
```
Comment (avdhoottt) [inline content/javascript/concepts/statements/statements.md:100]:
```suggestion

## Codebyte Example
```

File: content/javascript/concepts/statements/statements.md [hunk 41]
```diff
@@ -48,6 +75,18 @@ x = 6;
...
+
+This example shows re-assignment of value to variable `x`:
+
+```codebyte/js
+var x = 5;
+console.log(x)
+x = 6;
+console.log(x);
+```
+
```
Comment (avdhoottt) [inline content/javascript/concepts/statements/statements.md:88]:
```suggestion
```

File: content/javascript/concepts/statements/statements.md [hunk 42]
```diff
@@ -39,6 +59,13 @@ Expression statements, such as function calls or assignments, evaluate an expres
 // Function call
 console.log('Hello World');
 ```
+## Codebyte Example
```
Comment (avdhoottt) [inline content/javascript/concepts/statements/statements.md:62]:
```suggestion

## Codebyte Example
```

**Review by avdhoottt — CHANGES_REQUESTED** [review]
I have suggested some changes, please make them.

**Comment by Nnamdingozi** [comment]
Hi @avdhoottt , @dakshdeepHERE , I have made all the suggested changes. Kindly go through and see if all the changes are reflected. @dakshdeepHERE kindly note that @avdhoottt suggested that I should change codebyte/js back to codebyte/javascript and Codebyte Example back to codebyte Example. Thanks a lot for your contributions

**Comment by Nnamdingozi** [comment]
HI @avdhoottt , This is a soft reminder that I have made all the changes you recommended. I am eagerly awaiting your review. Thanks a lot.

**Comment by dakshdeepHERE** [comment]
@Nnamdingozi let me notify him for the review.

File: content/javascript/concepts/statements/statements.md (hunk 2 above)
Comment (avdhoottt) [inline content/javascript/concepts/statements/statements.md:35]:
```suggestion
## Codebyte Example
```

File: content/javascript/concepts/statements/statements.md [hunk 43]
```diff
@@ -70,6 +124,25 @@ switch (x) {
     console.log('x is neither 1 nor 2');
 }
 ```
+This example shows outputs of switch statement:
```
Comment (avdhoottt) [inline content/javascript/concepts/statements/statements.md:127]:
```suggestion

This example shows the outputs of the switch statement:
```

File: content/javascript/concepts/statements/statements.md [hunk 44]
```diff
@@ -100,9 +198,37 @@ for (let i = 0; i < 5; i++) {
...
 function add(x, y) {
   return x + y;
 }
 ```
+
+## Codebyte Example
+
+This code example returns the value of the add function: 
+
+```codebyte/javascript
```
Comment (avdhoottt) [inline content/javascript/concepts/statements/statements.md:228]:
```suggestion
```codebyte/js
```

**Review by avdhoottt — APPROVED** [review]
LGTM 🚀

**Pull request merged**